src/test/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepositoryTest.java

```diff
@@ -437,4 +437,38 @@ public void startJobSkipsNullParameters() {
 
     assertThat(parameters, is(empty()));
   }
+
+  @Test
+  public void parameters_populated_from_environment() {
+    cause = makeCause(null);
+
+    ParameterDefinition parameterDefinition =
+        new StringParameterDefinition("sourceBranch", "DefaultBranch");
+    jobSetup(parameterDefinition);
+
+    List<ParameterValue> parameters = captureBuildParameters();
+
+    assertThat(parameters, hasSize(1));
+    assertThat(parameters.get(0), instanceOf(StringParameterValue.class));
```

Comment:
I'm not sure about the need to check the type here. What are the chances it will be something else while satisfying the other assertions?

Reply:
Good point. Better yet, `contains()` takes case or it, as it requires constructing the expected value.